js/components/graph/Node.js

```diff
@@ -97,19 +143,23 @@ export default class Node extends React.Component {
     }
   }
 
+  /** Controls the selection and deselection of a node by switching states and updating the graph */
   toggleSelection = () => {
     this.setState({ selected: !this.state.selected }, function() {
       this.updateNode();
     });
   }
 
+  /** Sets the status of all missing prerequisites to 'missing' */
   focusPrereqs = () => {
-    var svg = this.props.svg;
-    // Check if there are any missing prerequisites.
+    var svg = this.props.svg;  // contains information about the whole graph
```

Comment:
Remove the inline comments

Reply:
The comment on line 157 was also not added by me, should I improve or delete it?

Reply:
Hmm, you can try to improve that one!

Reply:
I decided to improve it to answer "why",  like we did to the other one